frontend/packages/knative-plugin/src/components/sink-source/SinkSourceController.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { createModalLauncher, ModalComponentProps } from '@console/internal/components/factory';
+import SinkSource from './SinkSource';
+
+type SinkSourceControllerProps = {
+  obj: K8sResourceKind;
+};
+
+const SinkSourceController: React.FC<SinkSourceControllerProps> = (props) => {
+  const { obj } = props;
```

Comment:
nit: we could do this in one line
```suggestion
const SinkSourceController: React.FC<SinkSourceControllerProps> = ({obj, ...props}) => {
```